src/Features/Core/Portable/EditAndContinue/DebuggingSession.cs

```diff
@@ -9,15 +11,14 @@ namespace Microsoft.CodeAnalysis.EditAndContinue
     /// </summary>
     internal sealed class DebuggingSession
     {
-        private readonly Solution _initialSolution;
+        public readonly Solution InitialSolution;
+
+        private readonly Dictionary<ActiveInstructionId, LinePositionSpan> _activeStatementRemaps = new Dictionary<ActiveInstructionId, LinePositionSpan>();
```

Comment:
Where is `_activeStatementRemaps` used?

Reply:
Not used. Removed

---
In reply to: [163005595](https://github.com/dotnet/roslyn/pull/24316#discussion_r163005595) [](ancestors = 163005595)